src/NuGet.Services.AzureSearch/ISearchDocumentBuilder.cs

```diff
@@ -27,6 +27,11 @@ public interface ISearchDocumentBuilder
             SearchFilters searchFilters,
             long totalDownloadCount);
 
+        SearchDocument.UpdateIsExcludedByDefault UpdateIsExcludedByDefault(
```

Comment:
This is unused right? I think we only have `FullFromDb` knowing about this property today so this method shouldn't be needed.

When Auxiliary2AzureSearch wants to keep this information up to date, we will probably expand the `UpdateDownloadCount` to do both instead of having a separate document update.